src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3725,6 +3755,8 @@ void InitIntSolarDistribution(EnergyPlusData &state)
                         state.dataHeatBal->EnclSolQSWRad(enclosureNum) +=
                             state.dataHeatBalSurf->ZoneFractDifShortZtoZ(enclosureNum, OtherenclosureNum) *
                             (state.dataHeatBal->EnclSolQD(OtherenclosureNum) + sumSpaceQLTSW);
+                        state.dataHeatBal->EnclSolQSWRadLightsOtherZones(enclosureNum) +=
+                            state.dataHeatBalSurf->ZoneFractDifShortZtoZ(enclosureNum, OtherenclosureNum) * sumSpaceQLTSW;
```

Comment:
This doesn't look correct. This is multiplying the total SW radiation from lights in *this* `enclosureNum` times the fraction of diffuse shortwave coming from `OtherenclosureNum`.

Reply:
Hmmm...yeah.  Good catch.  I think the indices should be "reversed" in that equation and the initialization (line 3746) should be taken out of the loop and just zero out all before the loop.  Pending the discussion of the other comment...

Reply:
I'm not following.

Reply:
Ah, now that you mention it, I think there was a bit of double-counting and I think maybe that is what this was supposed to do--eliminate that.  It's vaguely familiar at this point, but that sounds right.  So, ignore my previous comment.  I wasn't remembering correctly what this was doing. 🤷‍♂️🤦‍♂️

I definitely agree with you, this is highly convoluted.  I think my goal was to get the clear bug fixed but not attempt any sort of clarification/rewriting of the code here.  I am concerned that any tinkering will lead to a much bigger project.  I'm not sure right before the release is a great time to jump into such an endeavor?

Reply:
So, instead of calculating `EnclSolQSWRadLightsOtherZones` and making a new output variable, how about simply adding this to `EnclSolQSWRadLights` here. Then you won't need other adjustment later, because the interzone lights will already be part of `EnclSolQSWRadLights`.

Reply:
And the reason for the apparent double-counting is that the actual surface heat balance works just has a single term for short-wave radiation (both solar and lights). The shortwave from lights is tracked just for reporting and then subtracted out of the total shortwave, again, just for reporting.

Reply:
@mjwitte Ok, I think I have successfully backed out the original solution and replaced it with the much cleaner alternative.  Code changes, revised unit test, and modified docs all included in the latest commits (also merged in develop) that I just pushed.  Hopefully this addresses your concerns and comes back green during the ci runs overnight...